### PR TITLE
GSoC ES translation

### PR DESCRIPTION
**Pull request by TheCoffeMaker**
```gsoc-2022.en.md```` translated to spanish in new file ```gsoc-2022.es.md```

Right now the flyer is pointing to english [EN] version, as [ES] version is not yet available. We can add the ES version and change it before merging the PR.

### CODE REVIEW

File: content/community/gsoc-2022.es.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+XSF y GSoC 2022 Logo](/images/logos/GSoC_2022_Logo.png)
+
+La XMPP Standards Foundation (XSF) tiene previsto participar como organización anfitriona en el [Google Summer of Code 2022](https://summerofcode.withgoogle.com/).
+Si estás interesado en participar como estudiante (o simplemente recién llegado al código abierto), mentor o tienes un proyecto XMPP, ¡Ésta es la página por la que debes empezar!.
+ 
+Si te imaginas trabajando a tiempo parcial (o incluso a tiempo completo) en un proyecto de software de código abierto durante el verano, entonces considera unirte a un proyecto de tecnología de comunicación en tiempo real XMPP y ¡participa en el Google Summer of Code 2022!
+
+### XMPP - el Protocolo Extensible de Mensajería y Presencia
```
Comment (Echolon) [inline content/community/gsoc-2022.es.md:12]: can we keep the part in English?

XMPP - el Extensible Messaging and Presence Protocol

File: content/community/gsoc-2022.es.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+XMPP es el Protocolo Extensible de Mensajería y Presencia, un conjunto de tecnologías abiertas para la mensajería instantánea, la presencia, el chat multipartito, las llamadas de voz y vídeo, la colaboración, el middle-ware ligero, la federación de contenidos y el enrutamiento generalizado de datos XML. XMPP se desarrolló originalmente en la comunidad de código abierto Jabber para ofrecer una alternativa abierta y descentralizada a los servicios de mensajería instantánea cerrados de la época.
```
Comment (Echolon) [inline content/community/gsoc-2022.es.md:14]: same here (you can put it in brackets translated though)

**Review by Echolon — CHANGES_REQUESTED** [review]
Great work guys! @TheCoffeMaker

**Comment by Echolon** [comment]
@TheCoffeMaker 

Under the title, can you add something in italic format alike:

Para que conste, la comunicación principal sobre el tema está en inglés y sólo la información en inglés se considera oficial.

Compare: https://xmpp.org/de/community/gsoc-2022/

Sorry I forgot this

File: content/community/gsoc-2022.es.md (hunk 1 above)
Comment (TheCoffeMaker) [reply content/community/gsoc-2022.es.md:12]: Sure,but like I said in the MUC ... better if its all the way in english

**Review by Echolon — APPROVED** [review]
Great thing!

**Comment by Echolon** [comment]
Muchas gracias todos!

**Pull request merged**